docs/notebooks/pre_executed/test_snia.ipynb

```diff
@@ -1,5 +1,13 @@
 {
  "cells": [
+  {
```

Comment:
We can remove this notebook altogether. It has been replaced by the plastic_snia.ipynb notebook.